Price chart - initial functions

mariarabin opened the pull request:
working code for the different time chart display (code to be enhanced still)

mariarabin committed:
Added priceChart.js & priceChart.html files

mariarabin committed:
Merge branch 'Initial'

mariarabin committed:
Merge branch 'master' of github.com:ayang726/Project-2

mariarabin committed:
Merge branch 'master' into price-chart

# Conflicts:
#	app/routes/htmlRoutes.js

mariarabin committed:
Update the price chart.html

mariarabin committed:
Merge branch 'master' into price-chart

mariarabin committed:
Fixed server.js head

mariarabin committed:
Draft of charting

mariarabin committed:
Updated  volume of price chart html file

mariarabin committed:
Added the 1-Month Function

mariarabin committed:
Updated all the functions

mariarabin committed:
Created a priceChart.js files for all the chart functions

alexjzyang commented on app/routes/htmlRoutes.js:
```diff
@@ -0,0 +1,12 @@
+// const path = require("path");
```
app folder doesn't exist anymore, Could you please incorporate your changes in the routes folder in the root folder?
Thanks

alexjzyang commented on public/js/priceChart.js:
```diff
@@ -0,0 +1,315 @@
+var dataSets = [];
```
I see you've installed chart.js as a dependancy to the node project. this means we can just require it in the priceChart js file.

mariarabin replied:
priceChart.js file si purely all my JS functions - so I did not add it.

alexjzyang commented on public/js/priceChart-package.js:
```diff
@@ -0,0 +1,7 @@
+/*!
```
We don't need this file anymore because we installed it as a node dependancy

alexjzyang commented on public/home.html:
```diff
@@ -1,16 +1,63 @@
\ No newline at end of file
+</html>
+
+<script src="/js/priceChart-package.js"></script>
```
Same here, we don't need to include priceChart-package.js because we installed it to Node

mariarabin committed:
Merge branch 'master' into price-chart

mariarabin committed:
1) Removed the app folder
2) Updated the home.html for the pricechart display
3) renamed the priceChart-package.js to be priceChart-reset.js
4) added the home.html src link in the home.handlebars

ayang726 commented:
looking good Maria~
